cumulusci.yml

```
@@ -87,6 +87,7 @@ tasks:
             output: robot/EDA/doc/Keywords.html
 
     update_admin_profile:
+        description: Sets record type visibilities for System Administrative profile
```

I'd like to not overwrite the built-in description. Can we add to it instead? Suggested:

```suggestion
        description: Retrieves, edits, and redeploys the System Administrator profile with full FLS perms for all objects/fields. Also sets record type visibilities.
        group: Custom Tasks
```

Also adding `group:` here to make clear that we've overridden the default task.